cymetric/metrics.py

```diff
@@ -322,6 +326,47 @@ def fco_u_mined(series):
 del _udeps, _uschema
 
 
+# SWU Required [million SWU]
+_swudeps = [
+    ('Materials', ('ResourceId', 'ObjId', 'TimeCreated', 'NucId'), 'Mass'),
+    ('Transactions', ('ResourceId',), 'Commodity')
+    ]
+
+_swuschema = [('Year', ts.INT), ('SWU', ts.DOUBLE)]
+
+@metric(name='FcoSwu', depends=_swudeps, schema=_swuschema)
+def fco_swu(series):
+    """FcoSwu metric returns the separative work units required for each 
+    year in a 200-yr simulation. This is written for FCO databases that 
+    use the Bright-lite (i.e., the U235 and U238 are given separately 
+    in the FCO simulations).
+    """
+    tools.raise_no_pyne('SWU Required could not be computed', HAVE_PYNE)
+    mass = pd.merge(series[0].reset_index(), series[1].reset_index(),
+            on=['ResourceId'], how='inner').set_index(['ObjId', 'TimeCreated', 'NucId'])
+    swu = []
+    prods = {}
+    mass235 = {}
+    m = mass[mass['Commodity'] == 'LWR Fuel']
+    for (obj, _, nuc), value in m.iterrows():
+        prods[obj] = prods.get(obj, 0.0) + value['Mass']
```

Comment:
Correct.  The LWR fuel recipe may contain any number of elements/isotopes and the definition of enrichment is U-235/(Total U).  Your current formula is (U-235)/(Total all elements)